src/v2/Apps/Conversation/Components/Message.tsx

```diff
@@ -59,23 +58,35 @@ interface AttachmentProps {
 
 export const Attachment: React.FC<AttachmentProps> = props => {
   const { attachment, alignSelf, bgColor, textColor } = props
+  const isImage = attachment.contentType.startsWith("image")
 
   return (
-    <AttachmentLink href={attachment.downloadURL} target="_blank">
+    <AttachmentLink
+      href={attachment.downloadURL}
+      target="_blank"
+      isImage={isImage}
+    >
       <AttachmentContainer
         p={1}
         mt={0.5}
         alignSelf={alignSelf}
         background={color(bgColor)}
+        width={isImage ? "100%" : "min-content"}
```

Comment:
👆 👇  Fix for attachment image size: when the attachment is image take 100% of the parent component width, which is 66% percent of the total width.